api/interestr/api/serializers.py

```diff
@@ -18,23 +18,32 @@ class Meta:
 class GroupSerializer(serializers.ModelSerializer):
     data_templates = DataTemplateSerializer(many=True, read_only=True)
     tags = TagSerializer(many=True, read_only=True)
+    members = serializers.PrimaryKeyRelatedField(many=True, read_only=True)
+    moderators = serializers.PrimaryKeyRelatedField(many=True, read_only=True)
 
     class Meta:
         model = core_models.Group
-        fields = ('id', 'name', 'created', 'updated', 'size', 'members', 'moderators', 'picture', 'data_templates',
+        fields = ('id', 'name', 'created', 'updated', 'size', 'members', 'location', 'description', 'moderators', 'picture', 'data_templates',
```

Comment:
I remember saying the description field was not coming from the API, weeks ago. Quite glad that it is added now! :V